CC/include/CCTypes.h

```diff
@@ -23,6 +23,12 @@
 using PointCoordinateType = float;
 
 //! Type of a single scalar field value
+#if defined SCALAR_T_DOUBLE
+using ScalarType = double;
+#elif defined SCALAR_T_FLOAT
 using ScalarType = float;
+#else
+static_assert(false, "type for ScalarType has not been undeclared");
```

Comment:
Do you mean 'declared' instead of undeclared?

Reply:
🤦‍♂